### PR TITLE
[zh-*] Translate missing lines.

### PR DESCRIPTION
**Pull request by overtrue**

### CODE REVIEW

File: json/zh_CN.json [hunk 1]
```diff
@@ -1,175 +1,192 @@
...
+    "Disable": "禁用",
+    "Done.": "已完成。",
     "E-Mail Address": "E-mail",
-    "Editor users have the ability to read, create, and update." : "Editor users have the ability to read, create, and update.",
-    "Editor" : "Editor",
-    "Email Password Reset Link": "Email Password Reset Link",
+    "Editor users have the ability to read, create, and update.": "编辑者可以阅读、创建和更新。",
+    "Editor": "编辑者",
+    "Email Password Reset Link": "电子邮件密码重置链接",
     "Email": "Email",
```
Comment (caouecs) [inline json/zh_CN.json:51]: Can you add it in exclude.php ?

File: todo.md [hunk 2]
```diff
@@ -14721,458 +14721,41 @@ into your web browser:
...
- * json : Add additional security to your account using two factor authentication.
- * json : Add Team Member
- * json : Add
- * json : Added.
- * json : Administrator users can perform any action.
- * json : Administrator
- * json : All of the people that are part of this team.
- * json : Already registered?
- * json : API Token Permissions
  * json : API Token
```
Comment (caouecs) [inline todo.md:14726]: If it's same translation, add it in exclude.php

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you, just add translations in exclude directory

File: todo.md (hunk 2 above)
Comment (overtrue) [reply todo.md:14726]: OK

**Comment by caouecs** [comment]
Thank you

**Comment by overtrue** [comment]
@caouecs ❤️

**Pull request merged**